.travis.yml

```diff
@@ -1,12 +1,16 @@
+sudo: required
+dist: precise
+
 language: php
 php:
   - 5.5
-  - 5.6
+  - 7
 
-services:
-  - elasticsearch
+before_install:
+  - curl -O https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.2.0/elasticsearch-2.2.0.deb && sudo dpkg -i --force-confnew elasticsearch-2.2.0.deb
```

Comment:
@jrbasso I was trying to reduce the number of matrix for this, however this library tends to not get a lot of updates, so I'll scratch it and bring on the matrices (including the 5.6 from above).